comixed-release/src/main/assembly/scripts/run.sh

```diff
@@ -74,7 +74,7 @@ if $VERBOSE; then
   set -x
 fi
 
-if [[ -f "${COMIXED_JAR_FILE}" ]]; then
+if [[ ! -f "${COMIXED_JAR_FILE}" ]]; then
```

Comment:
I'm not sure how this ever worked:  `-f` tests "if file exists and is a regular file" (not a directory or device).  So the previous code was testing, "If JAR exists, exit because the JAR is missing."  I'm guessing it worked because the `*` was being treated like a literal, so by failing, it succeeded?

Reply:
That sounds right.